modules/tensor_mechanics/src/materials/ADGeneralizedRadialReturnStressUpdate.C

```diff
@@ -55,7 +57,11 @@ ADGeneralizedRadialReturnStressUpdate::ADGeneralizedRadialReturnStressUpdate(
         _base_name + getParam<std::string>("inelastic_strain_rate_name"))),
     _max_inelastic_increment(getParam<Real>("max_inelastic_increment")),
     _max_integration_error(getParam<Real>("max_integration_error")),
-    _max_integration_error_time_step(std::numeric_limits<Real>::max())
+    _max_integration_error_time_step(std::numeric_limits<Real>::max()),
+    _angle(isParamValid("rotation_angles") ? getParam<RealVectorValue>("rotation_angles")
```

Comment:
If these rotation angles are implying an Euler sequence (e.g. ZYX), it may be good to add it to the description.

Reply:
Done!